encoder/vaapiencoder_base.cpp

```diff
@@ -322,7 +326,7 @@ SurfacePtr VaapiEncoderBase::createSurface(const SharedPtr<VideoFrame>& frame)
 
 void VaapiEncoderBase::fill(VAEncMiscParameterHRD* hrd) const
 {
-    hrd->buffer_size = m_videoParamCommon.rcParams.bitRate * m_videoParamCommon.rcParams.windowSize/1000;
+    hrd->buffer_size = m_videoParamCommon.rcParams.bitRate * 4;
```

Comment:
vaapi defines window_size to indicate driver must guarantee target bit-rate during a window time, but original code used windowSize as HDR CBP buffer size in ms.
4 just means HDB buffer_size is 4 times of bitrate, you can set another value,such as 2,6,8. And it can be changed for every IDR frame, so I don't think it's necessary to use a macro.